data/modules/SearchRescue/SearchRescue.lua

```diff
@@ -727,7 +728,8 @@ local onChat = function (form, ref, option)
 			entity       = ad.entity,
 			problem      = ad.problem,
 			cash         = Format.Money(ad.reward),
-			ship         = shipdef.name,
+			shipname     = shipdef.name,				-- Ship name, base form ex. 'Malabar'
```

Comment:
This name is not translated.

For example, german:
`shipname` == "Sinonatrix (Police)"
`shipsing` == "ein Sinonatrix Polizeischiff"

Even in english there will be a difference, so somewhat confusing and disjointed player experience I think.